src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,63 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+const urlImage = 'https://m.media-amazon.com/images/'
```

Comment:
```suggestion
const defaultUrlImage = 'https://m.media-amazon.com/images/'
```
You can specify it, so other developers know the purpose of this variable